android/app/src/main/res/values-de/strings.xml

```diff
@@ -25,4 +25,17 @@
     <string name="error_no_city_found">Keine Stadt mit dem Namen \"%1$s\" gefunden.</string>
     <string name="user">Benutzer</string>
     <string name="search">Suche</string>
+    <string name="add">Hinzufügen</string>
+    <string name="hours_unit">%1$s std</string>
+    <string name="rate">Bewerten</string>
+    <string name="number_of_stop_found">%d Sehenswürdigkeiten gefunden!</string>
+    <string name="category_place_of_worship">Gotteshaus</string>
```

Comment:
Maybe something like "Religiöse Versammlungsstätte"... I mean there are religions that are praying to the pasta monster....